lib/setup/SetupClass.php

```diff
@@ -711,7 +746,7 @@ public function drop()
         }
         foreach ($aDropTables as $sDrop) {
             if ($this->bVerbose) echo "Dropping table $sDrop\n";
-            @pg_query($this->oDB->connection, "DROP TABLE $sDrop CASCADE");
+            $this->oDB->exec('DROP TABLE $sDrop CASCADE');
```

Comment:
You can't replace " with ' here. There is a variable in the string.

Reply:
good catch